pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+   Any meta-POP is also considered a Process POP.
+
+
+POP Workflow
+============
+
+
+PyLadies's BDFL
+-------------
+
+There are several reference in this POP to the "BDFL". This acronym stands
+for "Benevolent Dictator for Life" and presently refers to Lynn Root, the
+current PyLadies liason to the PSF.
+
+
+POP Editors
+-----------
+
+The POP editors are individuals responsible for managing the administrative
+and editorial aspects of the POP workflow (e.g. assigning POP numbers and
+changing their status).  See `POP Editor Responsibilities & Workflow`_ for
+details.  The current editors are:
+
+* Selena Deckelman
+* Jackie Kaziel
+* Esther Nam
+* Audrey Roy
+* Carol Willing
+
+POP editorship is by invitation of the current editors.  The address
+<POPs@PyLadies.org> is a mailing list for contacting the POP editors.  All
+email related to POP administration (such as requesting a POP number
+or providing an updated version of a POP for posting) should be sent to
+this address (no cross-posting please).
+
+
+Submitting a POP
+----------------
+
+The POP process begins with a new idea for PyLadies.  It is highly
+recommended that a single POP contain a single key proposal or new
+idea. Small enhancements or patches often don't need
+a POP and can be injected into the PyLadies development workflow with a
+patch submission to the PyLadies `issue tracker`_. The more focused the
+POP, the more successful it tends to be.  The POP editors reserve the
+right to reject POP proposals if they appear too unfocused or too
+broad.  If in doubt, split your POP into several well-focused ones.
+
+Each POP must have a champion -- someone who writes the POP using the
+style and format described below, shepherds the discussions in the
+appropriate forums, and attempts to build community consensus around
+the idea.  The POP champion (a.k.a. Author) should first attempt to
+ascertain whether the idea is POP-able.  Posting to the
+comp.lang.PyLadies newsgroup (a.k.a. PyLadies-list@PyLadies.org mailing
+list) or the PyLadies-ideas mailing list is the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a POP is meant
+to save the potential author time. Many ideas have been brought
+forward for changing PyLadies that have been rejected for various
+reasons. Asking the PyLadies community first if an idea is original
+helps prevent too much time being spent on something that is
+guaranteed to be rejected based on prior discussions (searching
+the internet does not always do the trick). It also helps to make sure
+the idea is applicable to the entire community and not just the author.
+Just because an idea sounds good to the author does not
+mean it will work for most people in most areas where PyLadies is used.
+
+Once the champion has asked the PyLadies community as to whether an
+idea has any chance of acceptance, a draft POP should be presented to
+PyLadies-ideas.  This gives the author a chance to flesh out the draft
```

Comment:
This is where we should include the bit to create a Pull Request against this repo.